request-handler/src/main/java/au/org/aodn/aws/wps/operation/DescribeProcessOperation.java

```diff
@@ -34,52 +36,86 @@ public DescribeProcessOperation(DescribeProcess request) {
         this.request = request;
     }
 
+    public DescribeProcess getRequest()
+    {
+        return this.request;
+    }
+
     @Override
     public String execute()
     {
-        String processDescriptionsS3Bucket = WpsConfig.getConfig(DESCRIBE_PROCESS_S3_BUCKET_CONFIG_KEY);
-        String S3KeyPrefix = WpsConfig.getConfig(DESCRIBE_PROCESS_S3_KEY_PREFIX_CONFIG_KEY);
-        String s3RegionName = WpsConfig.getConfig(AWS_REGION_CONFIG_KEY);
+        String processDescriptionsS3Bucket = WpsConfig.getConfig(WpsConfig.DESCRIBE_PROCESS_S3_BUCKET_CONFIG_KEY);
```

Comment:
Would a static import make these constant references cleaner/easier to read?

Reply:
Potato/Potato  :)